random.js

```diff
@@ -1,9 +1,96 @@
-var jugadores = ["ignacio","victor","nil","camaleonte","diodoro","rober","lozano","pepe","adrian","ramon","jimeno","carlos"];
-var dado = Math.floor(Math.random() * jugadores.length);
-var mensaje = "<h1>"+jugadores[dado]+"</h1>";
-document.write(mensaje);
-
-var jugadores = ["ignacio","victor","nil","camaleonte","diodoro","rober","lozano","pepe","adrian","ramon","jimeno","carlos"];
```

Comment:
Add a space after commas:
```
var jugadores = ["ignacio", "victor", "nil", "camaleonte"]
```
Programming style is something very personal, but there's some generally accepted rules, and it's good to be consistent with all the code in the same project.